src/test/unit/inboundApi/handlers.test.js

```diff
@@ -93,17 +96,48 @@ describe('Inbound API handlers:', () => {
             await expect(handlers['/transactionRequests'].post(mockTransactionReqContext)).resolves.toBe(undefined);
 
             expect(transactionRequestSpy).toHaveBeenCalledTimes(1);
-            expect(transactionRequestSpy.mock.calls[0][0]).toBe(mockTransactionReqContext.request.body);
-            expect(transactionRequestSpy.mock.calls[0][1]).toBe(mockTransactionReqContext.request.headers['fspiop-source']);
+            expect(transactionRequestSpy).toHaveBeenCalledWith(mockTransactionReqContext.request.body,
+                mockTransactionReqContext.request.headers['fspiop-source']);
+        });
+    });
+
+    describe('POST /authorizations', () => {
+
+        let mockAuthorizationContext;
+
+        beforeEach(() => {
+
+            mockAuthorizationContext = {
+                request: {
+                    body: mockAuthReqArgs.authorizationRequest,
+                    headers: {
+                        'fspiop-source': 'foo'
+                    }
+                },
+                response: {},
+                state: {
+                    conf: {},
+                    logger: mockLogger({ app: 'inbound-handlers-unit-test' })
+                }
+            };
+        });
+
+        test('calls `ThirdpartyTrxnModelIn.postAuthorizations` with the expected arguments.', async () => {
+            const authorizationRequestSpy = jest.spyOn(ThirdpartyTrxnModelIn.prototype, 'postAuthorizations');
+
+            await expect(handlers['/authorizations'].post(mockAuthorizationContext)).resolves.toBe(undefined);
+            expect(authorizationRequestSpy).toHaveBeenCalledTimes(1);
+            expect(authorizationRequestSpy).toHaveBeenCalledWith(mockAuthorizationContext.request.body,
```

Comment:
please reformat to keep line width short

Reply:
Ok formatted now